### PR TITLE
Upgrade `puck` to `v0.15.0`

### PR DESCRIPTION
**Pull request by monospaced**

### CODE REVIEW

**Review by chrisvxd — CHANGES_REQUESTED** [review]
I think you need a body margin reset now

image

File: components/Heading/Heading.css [hunk 1]
```diff
@@ -5,91 +5,79 @@
...
-  font-weight: 700;
+  font: 700 var(--font-07);
   letter-spacing: var(--font-07-letter-spacing);
 }
 
 @media (min-width: 48em) {
   .msrd .msrd-Heading--2 {
-    font: var(--font-08);
+    font: 700 var(--font-08);
     font-weight: 700;
```
Comment (chrisvxd) [inline components/Heading/Heading.css:56]: Duplicate font-weight needs removing

File: components/Heading/Heading.css [hunk 2]
```diff
@@ -5,91 +5,79 @@
 }
```
Comment (chrisvxd) [inline components/Heading/Heading.css:5]: Paragraph styles weren't updated but also contain this issue

**Comment by monospaced** [comment]
@chrisvxd 

> I think you need a body margin reset now

There is one on the site, so that is a bit confusing. I think maybe it's the outer body in your screenshot that's coming from the site (as can also see `lang="en-gb" style={{ scrollBehavior: "smooth" }}` on the outer `html`). Is the `body` inside the frame not under puck control?

https://github.com/measuredco/measuredco-site/blob/main/app/layout.tsx#L54

File: components/Heading/Heading.css (hunk 2 above)
Comment (monospaced) [reply components/Heading/Heading.css:5]: And this 👍🏻

**Pull request merged**